client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domains.tsx

```diff
@@ -200,7 +200,7 @@ const Domains: React.FC< Props > = ( { onSubmit } ) => {
 			);
 		}
 
-		return __( 'Start transfer' );
```

Comment:
This was intentionally changed yesterday in a PR from @allilevine, so we should correct this.